client/src/lib/session/hooks/useUpdateSessionState.ts

```diff
@@ -39,22 +58,32 @@ const useUpdateSessionState = (
 
       const completed = index === content.length - 1 ? true : undefined;
 
-      return sessionApi.updateSessionState(sessionId, {
-        index,
-        playing: false,
-        completed,
-      });
+      try {
+        return await sessionApi.updateSessionState(sessionId, {
+          index,
+          playing: false,
+          completed,
+        });
+      } catch (error) {
+        showNetworkRequestError();
```

Comment:
I was thinking that this should be part of the `apiClient` as a general solution instead of implemented at each place, but with the option to no show errors. E.g. KillSwitch or metrics. WDYS?